core/test/common/FixedPoint.js

```diff
@@ -26,9 +26,19 @@ contract("FixedPoint", function(accounts) {
     assert.isFalse(await fixedPoint.wrapIsLessThan(web3.utils.toWei("2"), web3.utils.toWei("2")));
     assert.isTrue(await fixedPoint.wrapIsLessThan(web3.utils.toWei("2"), web3.utils.toWei("3")));
 
+    assert.isTrue(await fixedPoint.wrapIsEqual(web3.utils.toWei("1"), web3.utils.toWei("1").toString()));
```

Comment:
@mrice32 this my attempt to test the new `isEqual` with a `uint` as the second input, but I'm definitely not confident that this is the correct way to test it

Reply:
You'll need to add `wrapMixedIsEqual()` to test it. Look at `wrapMixedIsGreaterThan()` for an example.

Reply:
done